packages/shared/src/components/PostOptionsMenu.tsx

```diff
@@ -59,14 +56,14 @@ import { ActiveFeedContext } from '../contexts';
 import { useAdvancedSettings } from '../hooks/feed';
 import { useFeature } from './GrowthBookProvider';
 import { feature } from '../lib/featureManagement';
-import { ContextMenuDrawer } from './drawers/ContextMenuDrawer';
-import { RootPortal } from './tooltips/Portal';
 import { SourceSubscribeExperiment } from '../lib/featureValues';
+import { ContextMenu as ContextMenuTypes } from '../hooks/constants';
+import useContextMenu from '../hooks/useContextMenu';
 import { SourceType } from '../graphql/sources';
 import { withExperiment } from './withExperiment';
 
-const PortalMenu = dynamic(
-  () => import(/* webpackChunkName: "portalMenu" */ './fields/PortalMenu'),
+const ContextMenu = dynamic(
+  () => import(/* webpackChunkName: "ContextMenu" */ './fields/PortalMenu'),
```

Comment:
Very minor, but it's a common pattern we keep
Second part ofcourse when we change filename :) 
```suggestion
  () => import(/* webpackChunkName: "contextMenu" */ './fields/ContextMenu'),
```